packages/lab/src/input-next/InputNext.tsx

```diff
@@ -188,20 +148,25 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
         },
         classNameProp
       )}
+      ref={ref}
       style={inputStyle}
       {...other}
     >
       <input
+        aria-label={clsx(
+          a11yProps?.["aria-labelledby"],
+          inputProps["aria-labelledby"]
+        )}
         id={id}
         className={clsx(withBaseName("input"), inputProps?.className)}
         disabled={isDisabled}
         readOnly={isReadOnly}
-        ref={ref}
```

Comment:
I think we might want to add an `inputRef` in case there is a need to ref the actual input. I believe input is a little different from checkbox and radio button. Not sure. Maybe we should add it as an enhancement later if there will be a need for it

Reply:
cc @joshwooding thoughts?

Reply:
I think inputRef might be needed